smoke-tests/smoke-sdk-grpc.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+load test_helpers/utilities
+
+CONTAINER_NAME="app-sdk-grpc"
+COLLECTOR_NAME="collector"
+TRACER_NAME="my-app"
+
+setup_file() {
+	echo "# 🚧" >&3
+	docker-compose up --build --detach collector
+	wait_for_ready_collector ${COLLECTOR_NAME}
```

Comment:
this differs from our other smoke tests because this app is so small and just runs and exits; this ensures the collector is ready before the app runs.